src/Microsoft.Identity.Web/Constants/IDWebErrorMessage.cs

```diff
@@ -36,6 +38,14 @@ internal static class IDWebErrorMessage
         public const string BlazorServerUserNotSet = "IDW10206: Using Blazor server but the user was not properly set. ";
         public const string CalledApiScopesAreNull = "IDW10207: The CalledApiScopes cannot be null. ";
         public const string ScopesRequiredToCallMicrosoftGraph = "IDW10208: You need to either pass-in scopes to AddMicrosoftGraph, in the appsettings.json file, or with .WithScopes() on the Graph queries. See https://aka.ms/ms-id-web/microsoftGraph. ";
+        public const string ApiAllowsAppOnlyTokensCantBeComputed = "IDW10209: You set the 'ApiAllowsAppOnlyTokens' configuration to false, but didn't opt-in to the 'idtyp' optional claim in the app registration. See https://aka.ms/ms-id-web/webapi-authorization. ";
+        public const string ApiDoesNotAllowsAppOnlyTokens = "IDW10210: This web API does not allow app-only tokens. ";
+        public const string ApiAllowsGuestAccountsCantBeComputed = "IDW10211: You set the 'ApiAllowsGuestAccounts' configuration to false, but didn't opt-in to the 'acct' optional claim in the app registration. See https://aka.ms/ms-id-web/webapi-authorization. ";
```

Comment:
what is the resolution for this, if i get this error?

Reply:
App needs to opt in to the acct claim in the optional claims section of the app reg